index.js

```diff
@@ -0,0 +1,163 @@
+/******* Deliverable 1 *******/
+// When the application loads, display all the information about the first player on the page, including name, nickname, photo, likes and goals.
+
+/*********** DOM Elements *********/
+const playerContainer = document.querySelector('.player')
+
+/*********** FETCH *********/
+// fetch("http://localhost:3000/players/1")
+// .then(response => response.json())
+// .then(data => renderPlayer(data))
+
+
+/********** Initial Render ********/
+const playerLikes = document.querySelector(".likes")
+const playerGoals = document.querySelector("#goals")
+const likesButton = document.querySelector('.like-button')
+const goalForm = document.querySelector('#new-goal-form')
+const goalList = document.querySelector("#goals")
+const formBtn = document.querySelector("#formButton")
+
+
+
+
+function renderPlayer(playerObj) {
+    const playerh2 = document.querySelector(".player h2")
+    const playerem = document.querySelector(".player em")
+    const playerImage = document.querySelector(".player img")
+     
+    
+    playerh2.textContent = playerObj.name
+    playerem.textContent = playerObj.nickname
+    playerImage.src = playerObj.photo
+    playerLikes.textContent = playerObj.likes 
+    
+    playerObj.goals.forEach(function(goal) {
```

Comment:
It'd be good to reuse the `renderGoals` function here too, so you don't end up duplicating code.